src/main/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTask.java

```diff
@@ -109,6 +110,11 @@
     /** The maximum number of warnings that are tolerated. */
     private int maxWarnings = Integer.MAX_VALUE;
 
+    /** The checker threads number. */
+    private int checkerThreadsNumber = 1;
+    /** The tree walker threads number. */
+    private int treeWalkerThreadsNumber = 1;
```

Comment:
@soon and @sabaka , If there is no answer to my question ... please create issue for support of MT in ANT and move that fix to separate branch. If changes are not required right now - they should be postponed.